### PR TITLE
Update SetupTemplates.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/modules/SetupTemplates.py [hunk 1]
```diff
@@ -550,11 +550,13 @@ def HFSS3DLayout_AdaptiveFrequencyData(freq):
     Parameters
     ----------
     freq : float
-        Frequency value.
+        Value for frequency.
```
Comment (akaszynski) [inline pyaedt/modules/SetupTemplates.py:553]:
```suggestion
        Value for adaptive frequency.
```

File: pyaedt/modules/SetupTemplates.py [hunk 2]
```diff
@@ -550,11 +550,13 @@ def HFSS3DLayout_AdaptiveFrequencyData(freq):
...
         
 
     Returns
     -------
+    float
+        Value for frequency.
```
Comment (akaszynski) [inline pyaedt/modules/SetupTemplates.py:559]:
```suggestion
    list
        Frequency parameters.
```

**Review by akaszynski — APPROVED** [review]
Approved with minor suggestions.

File: pyaedt/modules/SetupTemplates.py [hunk 3]
```diff
@@ -942,21 +965,25 @@ def update(self):
...
             ``True`` when successful, ``False`` when failed.
+            
         """
         self.oanalysis.EditSweep(self.setupname, self.name, self._get_args())
         return True
 
     @aedt_exception_handler
     def _get_args(self, props=None):
-        """
+        """Retrieve arguments.
```
Comment (PipKat) [inline pyaedt/modules/SetupTemplates.py:975]:
```suggestion
        """Retrieve properties.
```

File: pyaedt/modules/SetupTemplates.py [hunk 4]
```diff
@@ -1062,22 +1094,26 @@ def update(self):
...
+        
         """
         self.oanalysis.EditSweep(self.setupname, self.name, self._get_args())
 
         return True
 
     @aedt_exception_handler
     def _get_args(self, props=None):
-        """
+        """Retrieve arguments.
```
Comment (PipKat) [inline pyaedt/modules/SetupTemplates.py:1105]:
```suggestion
        """Retrieve properties.
```

**Pull request merged**